packages/client/src/components/domain/shop/ManufacturerProductListPage.tsx

```diff
@@ -32,6 +34,51 @@ export const ManufacturerProductListPage = () => {
   const { products } = useHandleProducts(manufacturerId!);
   const items = products?.products ?? [];
 
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const openModal = () => {
+    setIsModalOpen(true);
+  };
```

Comment:
[nits] 中括弧を省略するともっとシンプルになりそうです！

```suggestion
  const openModal = () => setIsModalOpen(true)
```

参考
https://typescriptbook.jp/reference/functions/arrow-functions#%E3%82%A2%E3%83%AD%E3%83%BC%E9%96%A2%E6%95%B0%E3%81%AE%E7%9C%81%E7%95%A5%E5%BD%A2